### ISSUE TITLE
Sorting not working anymore

### ISSUE THREAD

**Issue by kixune21** [newsboat/newsboat#1028]
Using version 2.20

Config file:

```
browser qutebrowser
color listnormal color5 default
color listfocus color0 color1
color listfocus_unread color0 color1
color info color0 color6 underline
auto-reload yes%  
```
Steps to reproduce the issue:

1. Enter to a feed list.

2. Press G or g for sorting and select any sorting method.

3. Sorting doesn't occur.

I think the problem may be realted to new translations, at least for the spanish language. Before this problem, sorting options were displayed in english, now they are in spanish and ask for different characters, but none of them seem to work.

**Comment by dennisschagt** [comment]
It looks like the Spanish translation is missing one of the sort options (`(r)andom`).
https://github.com/newsboat/newsboat/blob/34294027b3c887e4d491cf2f2ecf42e2fb7ad04e/po/es.po#L960-L975

When the option string in the translated language is not equal to the original length (`len('dtfalgr') == 7 ≠ 6 == len('ftmaeg')`), the original key characters are used.
If you want, you can check this by using one of the letters shown in the English strings.

https://github.com/newsboat/newsboat/blob/34294027b3c887e4d491cf2f2ecf42e2fb7ad04e/src/itemlistformaction.cpp#L686-L694

**Comment by Minoru** [comment]
Thanks for the report, @kixune21, and thanks to @dennisschagt for the triage! I've submitted a pull request that fixes this, #1029, and I'll release 2.20.1 right after merging it.
